tests/unit/src/formatMarkup.test.js

```diff
@@ -249,6 +249,54 @@ describe('Format markup', () => {
     });
   });
 
+  describe('Self Closing Tags', () => {
+    let MyComponent;
+
+    beforeEach(() => {
+      MyComponent = {
+        template: '<div></div><span class="orange"></span><svg><path d=""></path></svg> <input></input> <input type="range"> <textarea></textarea>'
+      };
+      globalThis.vueSnapshots.formatter = 'diffable';
+    });
+
+    test('Enabled', () => {
+      const wrapper = mount(MyComponent);
+      globalThis.vueSnapshots.formatting.selfClosingTag = true;
+      expect(wrapper)
+        .toMatchInlineSnapshot(`
+          <div />
+          <span class="orange" />
+          <svg>
+            <path d="" />
+          </svg>
+          <input value="''" />
+          <input
+            type="range"
+            value="''"
+          />
+          <textarea value="''"></textarea>
+        `);
+    });
```

Comment:
Might be good to explicitly set `voidElements` too, to better convey why they are included in `MyComponent` (to show they are not impacted by this setting.
```suggestion
      globalThis.vueSnapshots.formatting.selfClosingTag = true;
      globalThis.vueSnapshots.formatting.voidElements = 'html';

      expect(wrapper)
        .toMatchInlineSnapshot(`
          <div />
          <span class="orange" />
          <svg>
            <path d="" />
          </svg>
          <input value="''">
          <input
            type="range"
            value="''"
          >
          <textarea value="''"></textarea>
        `);
    });
```